Project.toml

```diff
@@ -11,9 +11,12 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 NLPModels = "a4795742-8479-5a88-8948-cc11e1c8c1a6"
 NLPModelsIpopt = "f4238b75-b362-5c4c-b852-0801c9a21d71"
 Symbolics = "0c5d862f-8b57-4792-8d23-62f2024744c7"
+SymbolicUtils = "d1185830-fcd6-423d-90d6-eec64667417b"
 
 [compat]
 ADNLPModels = "0.7"
 CTBase = "0.8"
 DocStringExtensions = "0.9"
+Symbolics = "5.28"
+SymbolicUtils = "1.5"
```

Comment:
@tmigot Julia will try to install the more recent version of `SymbolicUtils.jl`. Don't you need a `= 1.5` or an upper bound?

Reply:
Oh, you are right, thx